ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.8__amd64-stable.yaml

```diff
@@ -732,30 +732,29 @@ tests:
     - ref: idp-htpasswd
     - ref: openshift-extended-test-longduration
     workflow: cucushift-installer-rehearse-azure-ipi-private
-- as: azure-ipi-private-p2-f360
-  cron: 32 0 14 5 *
+- as: azure-ipi-private-workers-marketimage-p2-f360
+  cron: 44 15 8 6 *
   steps:
-    cluster_profile: azure-qe
+    cluster_profile: azure-marketplace-qe
     env:
       BASE_DOMAIN: qe.azure.devcluster.openshift.com
-      E2E_RUN_TAGS: '@azure-ipi and @disconnected and @network-openshiftsdn and not
-        @fips'
+      E2E_RUN_TAGS: '@azure-ipi and @network-openshiftsdn and not @fips'
       TAG_VERSION: '@4.8'
     test:
     - chain: openshift-e2e-test-qe
-    workflow: cucushift-installer-rehearse-azure-ipi-private
-- as: azure-ipi-private-p3-f360-destructive
-  cron: 14 9 26 3 *
+    workflow: cucushift-installer-rehearse-azure-ipi-private-workers-marketimage
+- as: azure-ipi-private-workers-marketimage-p3-f360-destructive
```

Comment:
I guess we can drop the destructive testing now per our latest stable test strategy.

Reply:
I didn't see the change on 4.8, and testrun `openshift-e2e-test-qe` is still there for other jobs, do I need to drop destructive job here?

Reply:
removed

Reply:
Yeah, @liangxia is update the existing one in another PR.

Reply:
got it